recipes/betse/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set name = "betse" %}
+{% set version = "0.7.0" %}
+{% set sha256 = "845896f4c30b3970d5bd9501e0ca754eef5af0d742de76103f66c8c8c58e4eca" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  entry_points:
+    - betse = betse.__main__:main
+  script: python -m pip install --no-deps --ignore-installed .
+  skip: True  # [py<34]
+
+requirements:
+  build:
+    - pip
+  host:
```

Comment:
I don't know if we support host currently.

Reply:
thanks for stepping in @CJ-Wright. Do you think this is why the linter fails?

Reply:
judging from [this example](https://github.com/conda-forge/staged-recipes/pull/5581), seems like this is the reason. The linter should probably give a more descriptive error message.

Reply:
Well, that's a relief. I'm delighted the fix is trivial, but sorrowful that it was ultimately my fault. I blame `conda skeleton pip`. That's the first and last time I run *that* particular command.

Thanks to the power of ∞, @CJ-Wright and @carlodri.

Reply:
I don't know if I'd throw the too out completely, we just haven't moved to conda build 3 yet.